cnd/src/btsieve/bitcoin/mod.rs

```diff
@@ -194,7 +232,7 @@ mod tests {
         let transaction = r#"02000000014135047eff77c95bce4955f630bc3e334690d31517176dbc23e9345493c48ecf000000004847304402200da78118d6970bca6f152a6ca81fa8c4dde856680eb6564edb329ce1808207c402203b3b4890dd203cc4c9361bbbeb7ebce70110d4b07f411208b2540b10373755ba01feffffff02644024180100000017a9142464790f3a3fddb132691fac9fd02549cdc09ff48700a3e1110000000017a914c40a2c4fd9dcad5e1694a41ca46d337eb59369d78765000000
 "#.to_owned();
 
-        let bytes = decode_response::<bitcoin::Transaction>(transaction);
```

Comment:
Unrelated to the 10% review:

You are removing the most important 7 characters from this line: bitcoin.

Knowing, _which_ transaction is decoded here is massively important because you will be able to immediately narrow down further possible paths/areas in your brain of what to think about.
Having to scroll to the top of the `tests` block to do that is a huge burden IMO.

Reply:
Will leave the paths in.